.env.example

```
@@ -17,3 +17,5 @@ DB_PATH=services/ingestion/db/data.sqlite
 
 # ── Logging ──────────────────────────────────────────────────────
 LOG_LEVEL=INFO
+
+CORS_ORIGINS=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)